app/javascript/packs/selectors/usersSelector.js

```diff
@@ -0,0 +1,11 @@
+export function getName(state) {
```

Comment:
when would the state not have a user? I think rmi-users definitely should have their user info in state as well (mentioned above in your `rmi-user-initial-state`)

Reply:
put it in!